src/mesh/RadioInterface.h

```diff
@@ -31,6 +31,12 @@ typedef struct {
 
     /** The channel hash - used as a hint for the decoder to limit which channels we consider */
     uint8_t channel;
+
+    // Last byte of the NodeNum of the next-hop for this packet
+    uint8_t next_hop;
+
+    // Last byte of the NodeNum of the current relayer of this packet
+    uint8_t current_relayer;
```

Comment:
If I am fallowing the code correctly "current_relayer" indicates where the packet just came from. For me it took me a moment to understand this since "current" often implies something that is active or happening at the present moment. I would suggest something like "last_relay_node" for the variable name.

Reply:
I understand why this can be confusing, but it depends on whether you look at it from the perspective of the transmitter or the receiver. For the transmitter, it would then need to set the `last_relay_node` of a packet it is about to sent to itself, which is also a bit strange. I took 'current' because a node it currently preparing/transmitting/receiving/processing this packet. Or would it be better to just call it `relay_node`?

Reply:
I think I understand but I don't know that I have a full picture yet of the code. My suggestion would be to error on the side of how the variable is used in logic. My impression is that it mostly gets used from the perspective of what was the last relay node. For clarity, I would then suggest adding a comment when transmitting "We are/will be the last_relay_node".

Reply:
Maybe the part I don't understand is "a packet it is about to sent to itself".

Reply:
I'm sorry, I mean that it has to set the `last_relay_node` to its own NodeNum before transmitting that packet.